src/components/cuda_pw/common_defs.h

```diff
@@ -0,0 +1,36 @@
+/*
+  This file contains data structures visible to linux-cuda.c and cupti_profiler.h
+*/
+
+#ifndef __COMMON_DEFS_H__
+#define __COMMON_DEFS_H__
+
+#include <papi.h>
+
+typedef struct eventname_id_s {
+    char name[PAPI_2MAX_STR_LEN];
+    char desc[PAPI_2MAX_STR_LEN];
+    int gpu_id;
+    unsigned int evt_code;
+    unsigned int evt_pos;  // index of added event
+    int num_dep;
+    double value;
+    void * info;  // API specific details
+} event_rec_t;
+
+typedef struct event_name_list_s {
+    struct eventname_id_s *evts;
```

Comment:
Why do you use `struct eventname_id_s` instead of `event_rec_t`?

Reply:
True. Will change.